doc/cli/npm-run-script.md

```diff
@@ -11,10 +11,13 @@ npm-run-script(1) -- Run arbitrary package scripts
 This runs an arbitrary command from a package's `"scripts"` object.
 If no package name is provided, it will search for a `package.json`
 in the current folder and use its `"scripts"` object. If no `"command"`
-is provided, it will list the available top level scripts.
+is provided, it will list the available top level scripts. The `env` command
+can be used to list environment variables that will be available to the script
+at runtime. The `env` command can be overridden in your package but it is
```

Comment:
"is not recommended" is pretty strongly-worded for a feature that we're just now adding. ;) Either drop this sentence or replace it with something that explains why it might be a bad idea.